<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MultisigInputEntry</class>
 <widget class="QFrame" name="MultisigInputEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>136</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Sunken</enum>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <layout class="QHBoxLayout" name="transactionIdLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="transactionId">
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter a transaction id</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pasteTransactionIdButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+P</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="deleteButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/remove</normaloff>:/icons/remove</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="transactionIdLabel">
     <property name="text">
      <string>Transaction id:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>transactionId</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="transactionOutputLabel">
     <property name="text">
      <string>Transaction output:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="redeemScriptLabel">
     <property name="text">
      <string>Redeem script:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="buddy">
      <cstring>redeemScript</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="redeemScriptLayout">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QValidatedLineEdit" name="redeemScript">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="toolTip">
        <string/>
       </property>
       <property name="placeholderText">
        <string>Enter the redeem script of the address in the transaction output</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="pasteRedeemScriptButton">
       <property name="toolTip">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../bitcoin.qrc">
         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
       </property>
       <property name="shortcut">
        <string>Alt+A</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="transactionOutput"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QValidatedLineEdit</class>
   <extends>QLineEdit</extends>
   <header>qt/qvalidatedlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../bitcoin.qrc"/>
 </resources>
 <connections/>
</ui>
